testsuite/integration/pom.xml

```diff
@@ -308,6 +308,7 @@
         <dependency>
             <groupId>org.wildfly</groupId>
             <artifactId>wildfly-naming-client</artifactId>
+            <version>1.0.0.Beta2</version>
```

Comment:
We shouldn't override the wildfly-naming-client version here, as per:
https://github.com/wildfly-security-incubator/wildfly/commit/b403fffe44d4e75bdcd2696e201df28aa13f35ff